README.md

```diff
@@ -34,21 +34,71 @@ Refer to these files by adding a `<link>` and a `<script>` element into your HTM
 
 ### Install with NPM
 
-If you have `node` installed on your machine, you can use `npm` to install the Web Design Standards. In your `package.json` list the following under `dependencies`:
+If you have `node` installed on your machine, you can use `npm` to install the
+Web Design Standards.
 
-```json
-  "uswds": "git@github.com:18F/web-design-standards.git#v0.9.0"
+```
+cd path/to/project-using-npm
+npm install --save uswds
 ```
 
-On subsequent runs of `npm install`, the package will be installed in `node_modules` under `uswds`. The CSS will be *not* compiled after it is installed. Instead, you are able to compile it however it fits into your project.
+This will add the Web Design Standards as a dependency for your project and save
+it to your `package.json` file. The package will be installed in `node_modules`
+under `uswds`. The CSS file(s) are generated on the `npm-prepublish` hook and can
```

Comment:
I think this line was added to explain why the CSS file isn't in the repo. A user that clones the project won't know where the CSS file is since it's only generated when it's published to `npm`.